src/net.md

```diff
@@ -1123,6 +1124,99 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-rate-limit-exceeded]: #ex-rate-limit-exceeded
+<a name="ex-rate-limit-exceeded"/>
+## Handling the Github rate limit error condition
+
+[![reqwest-badge]][reqwest] [![hyper-badge]][hyper] [![cat-net-badge]][cat-net]
+
+This example uses the [GitHub API - Rate limiting], as an example of how to handle error conditions.
+
+
+```rust,no_run
+#[macro_use]
+extern crate error_chain;
+#[macro_use]
+extern crate hyper;
+extern crate reqwest;
+# extern crate rayon;
```

Comment:
You don't seem to be using the `rayon` crate in this example, so I would remove references to `rayon`.

Reply:
is using Rayon to do calls in parallel and reach the limit of API calls, that block of code is hidden as is this one.

Reply:
Ah yes I missed that `into_par_iter` on line 1179... my apologies!

Reply:
Was important to develop and test this code sample, and it's now up to you how do you want to present this.